examples/gcp/main.tf

```diff
@@ -0,0 +1,16 @@
+locals {
+  project_id = "{{YOUR_PROJECT_ID}}"
+}
+
+provider "google" {
+  project = local.project_id
+}
+
+module "bitmovin_cloud_connect" {
+  project_id = local.project_id
+  source     = "../../modules/gcp" # local reference when you clone the repository
+  # source     = "github.com/bitmovin/terraform-cloud-connect/blob/main/modules/gcp" # remote GitHub reference
```

Comment:
Unsure, what works best when you clone the repo. But the README refers to the GitHub repo directly.